index.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+var _               = require("lodash");
```

Comment:
ah, thanks for pointing that out, I did not know you can do that! That's awesome. I will make the change.